iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/iothub/setup/SendMessagesCommon.java

```diff
@@ -208,7 +208,16 @@ public SendMessagesTestInstance(IotHubClientProtocol protocol, AuthenticationTyp
 
         public void setup() throws Exception
         {
-            SSLContext sslContext = SSLContextBuilder.buildSSLContext(x509CertificateGenerator.getX509Certificate(), x509CertificateGenerator.getPrivateKey());
+            // Builds an SSL context instance that trusts the certs on the device's certificate store
+            // and has no private keys
+            SSLContext sslContext = SSLContextBuilder.buildSSLContext();
+            if (testInstance.authenticationType == SELF_SIGNED)
```

Comment:
Since the SSL context build above does not contain private keys, how are CA certificate signed clients authenticated?

Reply:
The SSL context instance without private keys is only used when constructing a SAS authenticated device

Reply:
Oh, interesting. Is this a java specific requirement where the SSL context needs to be constructed even for a sas authenticated client? Is this similar to .NET accessing the device's certificate store directly during TLS negotiation?

Reply:
And the tests we run for Java only ever used self-signed certs, so we don't need the below 'if' block to include CA signed